models/stg_linkedin.yml

```diff
@@ -111,9 +111,10 @@ models:
           CANCELED - Creative is permanently canceled.
 
       - name: click_uri
-        description: |
-          Required when creative type is TEXT_AD. Maximum character limit of 500 after URL encoding all special characters. 
-          Limit does not include the "https://" prefix.
+        description: The URL where the member should be redirected to, on clicking the text ad or spotlight ad.
+
+      - name: click_uri_type
+        description: Will either be `text_ad`, `spotlight`, or `null` to indicate the origin of the click_uri.
```

Comment:
I feel like `null` can mean the click_uri either originates from the old `click_uri` field or just another click uri ad type. Do you feel it's fine that this scenario isn't differentiated?

Reply:
`null` would essentially mean it always is originating from the old click uri. We currently don't support other click uri ad types at the moment (see FR #70).

In this case I feel it's fine to keep it as null because in the deprecated click_uri there is no way to determine the type. So I would rather provide no information than provide contradicting information regarding the deprecated field.

Reply:
I see, because of the way click_uri is a coalesce among `text_ad_landing_page, spotlight_landing_page,` and `click_uri` so if none are present click_uri would just be null regardless if it's located in another ad type.

That makes sense to me!